app/controllers/api/v1/contacts_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class ContactsController < Api::V1::ApiController
+      helper_method :user, :admins
+
+      def create
+        Contact.create!(contact_params.merge(user_id: user.id))
```

Comment:
You can add in the user model `has_many :contacts`, and then here `user.contacts.create!(...)` so it's more object-oriented.

Reply:
I was wondering about this, since I wont ever be using contact from the user itself, I didnt think it was necessary to specify` has_many :contacts` on user model. But also I wasn't sure I could do this, implying not connecting bidirectionally the relation between the two models.  Is this last thing a must? @gonzaM